internal/diagnose/errors.go

```diff
@@ -2,77 +2,11 @@ package diagnose
 
 import "errors"
 
-type ErrConnection struct {
-	innerErr error
-}
-
-func NewErrConnection(err error) ErrConnection {
-	return ErrConnection{
-		innerErr: err,
-	}
-}
-
-func (e ErrConnection) Error() string {
-	return e.innerErr.Error()
-}
-
-func (e ErrConnection) Unwrap() error {
-	return e.innerErr
-}
-
-type ErrValidation struct {
-	innerErr error
-}
-
-func NewErrValidation(err error) ErrValidation {
-	return ErrValidation{
-		innerErr: err,
-	}
-}
-
-func (e ErrValidation) Error() string {
-	return e.innerErr.Error()
-}
-
-func (e ErrValidation) Unwrap() error {
-	return e.innerErr
-}
-
-type ErrDiscovery struct {
-	innerErr error
-}
-
-func NewErrDiscovery(err error) ErrDiscovery {
-	return ErrDiscovery{
-		innerErr: err,
-	}
-}
-
-func (e ErrDiscovery) Error() string {
-	return e.innerErr.Error()
-}
-
-func (e ErrDiscovery) Unwrap() error {
-	return e.innerErr
-}
-
-type ErrPostEvent struct {
-	innerErr error
-}
-
-func NewErrPostEvent(err error) ErrPostEvent {
-	return ErrPostEvent{
-		innerErr: err,
-	}
-}
-
-func (e ErrPostEvent) Error() string {
-	return e.innerErr.Error()
-}
-
-func (e ErrPostEvent) Unwrap() error {
-	return e.innerErr
-}
-
-var ErrLicenseKey = errors.New("")
-var ErrInsightsInsertKey = errors.New("")
+var (
+	ErrConnection        = errors.New("there was an error connecting to New Relic platform. This is an indication that your firewall or proxy settings do not allow outbound traffic to the New Relic platform. To configure the use of an HTTP proxy, use the HTTP_PROXY and/or HTTPS_PROXY environment variables. For more details visit https://github.com/newrelic/newrelic-cli/blob/main/docs/GETTING_STARTED.md#using-an-http-proxy")
+	ErrValidation        = errors.New("there was a failure locating the data that was posted to New Relic. Please try again later or contact New Relic support. For real-time platform status info visit https://status.newrelic.com/")
+	ErrDiscovery         = errors.New("failed to detect your system's hostname. Please contact New Relic support.")
```

Comment:
Linter doesn't like the punctuation here
![image](https://user-images.githubusercontent.com/6722433/118702205-a0036100-b7e2-11eb-9121-9dde383036c7.png)

Reply:
fixed